lua/nvim-treesitter/parsers.lua

```diff
@@ -1123,6 +1123,15 @@ list.luau = {
   maintainers = { "@amaanq" },
 }
 
+list.m3u = {
+  install_info = {
+    url = "https://github.com/Freed-Wu/tree-sitter-m3u",
+    files = { "src/parser.c" },
+  },
+  filetype = "m3u",
```

Comment:
```suggestion
```
Not needed.

Reply:
Fixed.

Reply:
Vim uses the `hlsplaylist` filetype.

Reply:
Fixed.

Reply:
You didn't have to rename the parser...

Reply:
Not serious. It also fix this problem :smile:

Reply:
There was no problem… You just had to set `filetype = "hlsplaylist"`…

Reply:
In general, if the language has an official name, use that name for the grammar. The filetype is a Vim-only thing and a separate issue.